cmake/Platform/Windows/PackagingPreBuild.cmake

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+file(REAL_PATH "${CPACK_SOURCE_DIR}/.." _root_path)
+set(_cpack_wix_out_dir ${CPACK_TOPLEVEL_DIRECTORY})
+file(TO_NATIVE_PATH "${_root_path}/scripts/signer/signer.ps1" _sign_script)
+
+set(_signing_command
+    powershell.exe
+    -nologo
+    -ExecutionPolicy Bypass 
+    -File ${_sign_script}
+)
+
+message(STATUS "Signing executable files in ${_cpack_wix_out_dir}")
+execute_process(
+    COMMAND ${_signing_command} -exePath ${_cpack_wix_out_dir}
+    RESULT_VARIABLE _signing_result
+    ERROR_VARIABLE _signing_errors
+    OUTPUT_VARIABLE _signing_output
+    ECHO_OUTPUT_VARIABLE
+)
+
+if(NOT ${_signing_result} EQUAL 0)
```

Comment:
It is often the case that retries are needed due to the availability of dependent resources, specifically, the time server.  A signing script should continue to try signing all files even if one file fails. Additionally, a retry should not try to sign a file that has been already signed in a previous attempt

Reply:
I think this was meant for the `signer.ps1` script? I can add a retry to the signing action or check the state of the time server and select another one if it fails. There's a flag to replace any existing signature, but to clarify: This job is meant to run clean, so no artifacts should be present if ran again.

Reply:
Retry added